meta-mender-st-stm32mp/README.md

```diff
@@ -0,0 +1,68 @@
+# meta-mender-st-stm32mp
+
+Mender integration layer for STM32MP family of boards.
+
+The supported and tested boards are:
+
+- [STM32MP157C Discovery Kit](https://hub.mender.io/t/stm32mp157c-discovery-kit/1676)
+
+Visit the individual board links above for more information on status of the
+integration and more detailed instructions on how to build and use images
+together with Mender for the mentioned boards.
+
+## Dependencies
```

Comment:
all dependencies are on dunfell, and as we discussed at https://hub.mender.io/t/stm32mp1-yocto-kirkstone-issues/5331 the setup instructions should actually refer to the ST wiki. I can also create a new Mender Hub article and retire the old one, free to suggestions here and happy to help

Reply:
I added some changes to the readme. please check it.